src/synthesized_insight/metrics/base.py

```diff
@@ -83,6 +84,81 @@ def __call__(self, sr_a: pd.Series, sr_b: pd.Series) -> Tuple[Union[int, float,
         return self._compute_test(sr_a, sr_b)
 
 
+class TwoColumnMap(_Metric):
```

Comment:
Looks like we might be missing a base '_Series' class here. 

Just like Metrics which return a single value, there used to be a base class which had a call function that returns a pandas Series.

Reply:
Created two base classes DataFrameMatrix and TwoDataFrameMatrix which computes metrics on a single and two dataframes respectively, and return the result in the dataframe.

TwoColumnMap is returning Dataframe, hence changed its base class from _Metric to TwoDataFrameMatrix